.github/workflows/docker-compose.yml.mysqldbdump

```diff
@@ -16,7 +16,7 @@ services:
       retries: 10
   edxapp:
     image: edxops/edxapp:latest
-    command: bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform/ && paver update_db'
+    command: bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform/ && make migrate'
```

Comment:
The old call included updating requirements first. Is that something that was redundant here?

Reply:
I believe re-installing requirements here would be redundant, because I believe the image should have a reasonably up-to-date version of the requirements installed.